src/app/shared-services/db/migrations.ts

```diff
@@ -116,6 +116,35 @@ export const MIGRATIONS = [
   // add field 'start' to table points
   `ALTER TABLE points ADD COLUMN state TEXT`,
 
+  // add primary key 'id' to table inferendes
```

Comment:
These new migration-entries should append to the end of the list of migrations. On devices with the latest state of the app, this way not all migrations would execute.

Reply:
resolved in 18cf7d0